Formula/traefik.rb

```diff
@@ -0,0 +1,61 @@
+require "language/go"
+
+class Traefik < Formula
+  desc "Træfik, a modern reverse proxy"
+  homepage "https://traefik.io/"
+  url "https://github.com/containous/traefik/archive/v1.3.8.tar.gz"
```

Comment:
This should use the release tarball https://github.com/containous/traefik/releases/download/v1.3.8/traefik-v1.3.8.src.tar.gz

Reply:
Should i comment here instead of root level to communicate completion of requested changes?